bluepyopt/ephys/locations.py

```diff
@@ -460,12 +532,175 @@ def instantiate(self, sim=None, icell=None):
             self.set_sec_index(icell=icell)
         return super().instantiate(sim=sim, icell=icell)
 
+    def acc_label(self):
+        """Arbor label"""
+        raise EPhysLocAccException('%s not supported in Arbor.' %
+                                   type(self).__name__)
+
+
+class ArbLocation(Location):
+    """Arbor Location"""
+
+    pass
+
+
+class ArbSegmentLocation(ArbLocation):
+    """Segment in an Arbor morphology.
+    """
+
+    def __init__(self, name, segment, comment=''):
+        super().__init__(name, comment)
+        self.segment = segment
+
+    def instantiate(self, sim=None, icell=None):  # pylint: disable=W0613
+        """Find the instantiate compartment"""
+        raise EPhysLocInstantiateException(
+            '%s not supported in NEURON.' % type(self).__name__)
```

Comment:
This is copy-pasted 5 times, perhaps it can be moved to the super class.

Reply:
Thanks for pointing this out - fixed in 02a00fbbc40b6004c611538b46d64cbebe930f7d.

Reply:
Thanks